packages/graphql-server-core/src/runQuery.ts

```diff
@@ -94,6 +94,7 @@ function doRunQuery(options: QueryOptions): Promise<ExecutionResult> {
     logFunction({action: LogAction.request, step: LogStep.status, key: 'operationName', data: options.operationName});
 
     // if query is already an AST, don't parse or validate
+    // XXX: This refers the operations-store flow.
```

Comment:
i wanted to disable getting to runQuery with document parsed to eliminate this flow at all.
but i've seen the module-operation-store depends on that "feature".